crates/cli/src/lib.rs

```diff
@@ -140,6 +142,9 @@ struct Remove {
     /// Whether to bypass the remove prompt.
     #[clap(long)]
     yes: bool,
+    /// Whether to bypass the root check
+    #[clap(long)]
+    bypass_root_check: bool,
```

Comment:
Only need that option on unix systems

```suggestion
    #[cfg(unix)]
    /// Whether to bypass the root check
    #[clap(long)]
    bypass_root_check: bool,
```